### ISSUE TITLE
[staging] DOTween warnings

### ISSUE THREAD

**Issue by redwyre** [StrangeLoopGames/EcoIssues#15169]
```
<color=#0099bc><b>DOTWEEN ► </b></color>Max Tweens reached: capacity has automatically been increased from 500/50 to 1250/50. Use DOTween.SetTweensCapacity to set it manually at startup
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogWarning(Object)
DG.Tweening.Core.Debugger:LogWarning(Object, Tween)
DG.Tweening.Core.TweenManager:GetTweener()
DG.Tweening.DOTween:ApplyTo(DOGetter`1, DOSetter`1, T2, Single, ABSTweenPlugin`3)
DG.Tweening.DOTween:To(DOGetter`1, DOSetter`1, Single, Single)
DG.Tweening.DOTweenModuleUI:DOFillAmount(Image, Single, Single)
DeltaBar:Set(Object)
System.Action`1:Invoke(T)
System.Runtime.Remoting.Contexts.CrossContextDelegate:Invoke()
ViewBehavior:InitView(View, View, ViewMemberInfo, BSONObject, Int32)
ViewExtensions:AutoSetupUI(View, ISubscriptions, Transform, String, Action, Boolean, Transform)
WorkOrderUI:ViewInited()
ViewBehavior:InitView(View, View, ViewMemberInfo, BSONObject, Int32)
ViewBehavior:<InitView>b__19_0()
System.Runtime.Remoting.Contexts.CrossContextDelegate:Invoke()
Eco.Shared.View.View:NotifyChanged(String)
Eco.Shared.View.ViewManager:ReceiveViewUpdate(BSONArray)
Eco.Shared.Networking.ClientPacketHandler:ReceiveWhile(NetworkClient, Func`1)
System.Action`1:Invoke(T)
FramePlanner.PlannerGroup:OnUpdate()
FramePlanner.FramePlannerSystem:OnUpdate()
Unity.Entities.ComponentSystem:InternalUpdate()
Unity.Entities.ComponentSystemGroup:OnUpdate()
Unity.Entities.ComponentSystem:InternalUpdate()
UnityEngine.Experimental.LowLevel.UpdateFunction:Invoke()
```
and

```
<color=#ff7337><color=#0099bc><b>DOTWEEN ► </b></color> SAFE MODE ►</color> DOTween's safe mode captured 183 errors. This is usually ok (it's what safe mode is there for) but if your game is encountering issues you should set Log Behaviour to Default in DOTween Utility Panel in order to get detailed warnings when an error is captured (consider that these errors are always on the user side).
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogWarning(Object)
DG.Tweening.Core.DOTweenComponent:OnDestroy()
```

**Comment by mirasrael** [comment]
Need to investigate why we allocate more than 500 tweens.

**Comment by gzatravkin** [comment]
Dottween is used only in Deltabar - special animated bar, that is used in work orders icon. If you create fast 5 workorders it goes to 120 dottweens (which is because of every workorder have several deltabars, and there are several same workorders icons). 

I don't reproducied it this warning, but I supose it was possible because of several updates of progress in work orders (every update of progress creates new dotweens). 

I made change so when it updates deltabar, it will first kill old dotween tweaks. 
I did not changed initial capacity so it won't suppress warnings just in case we will have another problem with Dotween.
